var/spack/repos/builtin/packages/dire/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Dire(Package):
```

Comment:
You should extend `AutotoolsPackage` here

Reply:
I considered it, but it is not in face an autotools package.
```14:53:03 wdconinc@herakles /tmp/dire/DIRE-2.004 $ ls
bin  configure  include  INSTALL  main  Makefile  Makefile.inc  README  share  src  VERSION```